pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -135,7 +135,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 					{Name: "NodeLabel"},
 					{Name: "ServiceAffinity"},
 				},
-				"PostFilterPlugin": {{Name: "DefaultPodTopologySpread"}},
+				"PreScorePlugin": {{Name: "DefaultPodTopologySpread"}},
```

Comment:
please add a new test to this file (```TestComponentConfigV1Alpha1```) to verify that the conversion is working as intended.

Reply:
sure